### DISCUSSION TITLE
À

### DISCUSSION THREAD

**Discussion by pcelem** [revoxhere/duino-coin#1745] [Q&A]

**Comment by Jonny11bonk** [comment]
Same as before, you're not supposed to send your personal information such as e-mails here into issues or discussions. Also, this is an English-only repo, unless you're contributing with translations and need help with it. Either use a translator and contact Duino-Coin via e-mail or use the contact form at the main website. Please, delete your issue if you wish to not have your e-mail address getting abused by someone.
